lib/realtime/vehicle.ex

```diff
@@ -63,19 +87,136 @@ defmodule Realtime.Vehicle do
     }
   """
   @spec decode(term()) :: t()
-  def decode(%{} = json) do
+  def decode(%{} = json, opts \\ []) do
```

Comment:
`@spec` needs to be updated with the new options argument.

Reply:
It's funny what dialyzer misses sometimes.

Reply:
It's optional, so there are really two functions, only one of which has a `@spec`.